### ISSUE TITLE
How to use the docs Search component on custom Navbar or above the Sidebar

### ISSUE THREAD

**Issue by Coderamrin** [shuding/nextra#2848]
I want to use a custom navbar and I need to have the searchbar also. 
But it's not available without the default navbar. 
Is there any way we can enable the search bar on the sidebar or on the custom navbar?

![image](https://github.com/shuding/nextra/assets/52592047/c5f06df2-c39d-4504-b05a-1b403cdb89eb)

**Comment by meganide** [comment]
Did you find a solution yet?

**Comment by Coderamrin** [comment]
> Did you find a solution yet?

No, I don't think it's possible. So, we used the default navbar with the searchbar.

**Comment by dimaMachina** [comment]
It's not possible, but `<Search>` component was exported in Nextra 4 https://the-guild.dev/blog/nextra-4#pagefind-setup
